ddraw/IDirect3DDeviceX.cpp

```diff
@@ -2512,17 +2570,39 @@ void m_IDirect3DDeviceX::InitDevice(DWORD DirectXVersion)
 
 void m_IDirect3DDeviceX::ReleaseDevice()
 {
+
 	WrapperInterface->DeleteMe();
 	WrapperInterface2->DeleteMe();
 	WrapperInterface3->DeleteMe();
 	WrapperInterface7->DeleteMe();
 
+#if WITH_IMGUI
+	if (Config.Dd7to9)
```

Comment:
Do all other IMGUI code blocks (have to) check this config setting as well?

Reply:
I think Elisha made sure of that.